vcxagency-node/test/unit/storage/pgstorage-webhooks.spec.js

```diff
@@ -45,4 +45,32 @@ describe('storage', () => {
     expect(await storage.getAgentWebhook(A)).toBe('https://example.org')
     expect(await storage.getAgentWebhook(B)).toBe('http://localhost:9123')
   })
+
+  it('should return null if webhook was unset', async () => {
+    const A = uuid.v4()
```

Comment:
I like how thoroughly tested this func is, but way more tests can fit in here without all these comments and emty lines :)

Reply:
I've deleted commented but I'll keep empty lines. When possible, I like to split tests in 3 parts - setup, execute whats being test, verify results